tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -215,13 +279,31 @@ func areRequiredRulesPresent(secGroupRules []network.SecurityRule) bool {
 	return true
 }
 
+// getIpPass extracts instance IPAddress and password from instance rdp file.
+func getIpPass(vmName string) (string, string, error) {
+	vmCredentialPath := filepath.Join(artifactDir, "/", vmName)
+	rdpCmd, err := ioutil.ReadFile(vmCredentialPath)
+	if err != nil {
+		return "", "", fmt.Errorf("failed to read %s credentials with error: %v", vmName, err)
+	}
+
+	// this regex looks for the IP address pattern from vmCredentialPath.
+	// the sample vmCredentialPath looks like xfreerdp /u:xxxx /v:12.23.34.45 /h:1080 /w:1920 /p:'password1234'
+	// ipAddress will capture IPaddress 12.23.34.45
+	ipAddress := ipAddressPattern.FindString(string(rdpCmd))
+
+	// passwordPattern will capture "/p:'password1234'". In that first 3 characters are unnecessary.
+	// so password contains "password1234'"
```

Comment:
If you are discarding the first three characters, shouldn't this be `'password1234'`?